internal/cluster/refresh_status_preprocessor.go

```diff
@@ -89,9 +90,19 @@ func (r *refreshPreprocessor) preprocess(ctx context.Context, c *clusterPreproce
 	for _, condition := range r.conditions {
 		stateMachineInput[condition.id.String()] = condition.fn(c)
 	}
+	for _, validationResults := range validationsOutput {
+		sortByValidationResultID(validationResults)
+	}
 	return stateMachineInput, validationsOutput, nil
 }
 
+func sortByValidationResultID(validationResults []ValidationResult) {
```

Comment:
What do you think about moving it to utils. We use it in 2 places.

Reply:
Was literally thinking this while doing the review! It'd be nice to do it

Reply:
Even though `ValidationResult` looks similar is different type. There is different `validationID` type used for host is `models.HostValidationID` and for the cluster is `models.ClusterValidationID`. We can't easily unify those types.